Cryptopia/Cryptopia.swift

```diff
@@ -102,6 +102,27 @@ public struct Cryptopia {
         }
     }
     
+    public class CryptopiaOrder: Order {
```

Comment:
Rename CryptopiaOrder to just Order. Reason: Tt is already inside the struct, Cryptopia.
Also, Subclass Order from Cryptex.Order to avoid name collision after renaming.